### DISCUSSION TITLE
ACAP IO Port (Nativ SDK)

### DISCUSSION THREAD

**Discussion by Cacsjep** [AxisCommunications/acap-native-sdk-examples#139] [General] [answered]
How could i change the output port to active from acap nativ sdk, I know I could do it via vapix but how could i do this via C api ?

Thanks

**Comment by shreyasatwork** [comment]
Hi @Cacsjep, , thanks for your question, we'll get back to you when we had time to look in to this.

**Reply by shreyasatwork** [reply]
Hello again, as of now, we are not able to suggest you a way to do this. As most of the teams who work with such IOCTL calls (not sure if that would be recommended) are on vacation, it was hard to get hold of them to get an answer.

**Reply by pataxis** [reply, marked as answer]
There is no C API for I/O port control today. There might come a solution to this in the future but right now there is only VAPIX.

**Comment by Cacsjep** [comment]
Hi @shreyasatwork, 

thank u for asking your team, i will try via DBUS.

**Reply by pataxis** [reply]
Hi @Cacsjep , I removed the previous answer since it was easy to misinterpret. D-Bus APIs are today not something we support from ACAP applications.
